mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -150,3 +186,121 @@ function BalanceItemRow ({
     </ThemedTouchableOpacity>
   )
 }
+
+interface DFIBalanceCardProps {
+  utxo: WalletToken
+  token: WalletToken
+  onConvertPress: () => void
+  onSendPress: () => void
+  onUtxoVsTokenPress: () => void
+  isLight: boolean
+}
+
+function DFIBalanceCard (props: DFIBalanceCardProps): JSX.Element {
```

Comment:
Maybe split this out on a different component/file?

Reply:
updated